examples/images_contours_and_fields/image_transparency_blend.py

```diff
@@ -0,0 +1,77 @@
+"""
+=============================================
+Blend transparency with color with 2-D images
+=============================================
+
+Blend transparency with color to highlight parts of data with imshow.
+
+A common use for :func:`matplotlib.pyplot.imshow` is to plot a 2-D statistical
+map. In this case, it's common to visualize the statistic of choice (e.g.,
+a t-statistic) alongisde another value of interest (e.g., the p-value for that
+statistic). One way to do this is to map the p-value onto the transparency of
+the image such that data with "significant" values are highlighted.
```

Comment:
I found this description a bit mysterious.  I'm not clear what is supposed to be "alongside" what.  I thought at first you were referring to the x and y axes, but I don't think you are.   

I'd simply say that its common to want to indicate significance of data values (or another quantity) using transparency overlain on an image.